check_prs.py

```
@@ -0,0 +1,23 @@
+import json, subprocess, os
+
+os.chdir(r"C:\Users\rahul\.gemini\antigravity\playground\vector-meteoroid\qwed-a2a")
+
+result = subprocess.run(
+    ["gh", "pr", "list", "--json", "number,title,headRefName,baseRefName,state,statusCheckRollup,reviews", "--limit", "20"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)